src/shapes.ts

```diff
@@ -91,6 +91,16 @@ export interface OpenFinWindow {
     id: number;
     name: string;
     uuid: string;
+    preloadState?: { // set if window has a preload script
+        src: String;
+        state:
+            'idle'| // initial state, not emitted, really here for internal use
+            'load-started'| // started loading/downloading the script file
```

Comment:
The state string is fine, but in reference to you comment: I think by "loading" here you mean when it's a `file://` scheme. However, I am using the term "load" differently, as in "fetch and load" where "fetch" gets the script into the cache (regardless of scheme); and "load" reads the cached file into memory. For consistency's sake, can we just say "fetch" instead of "load/download" ?